go/src/meguca/db/init.go

```diff
@@ -258,6 +258,13 @@ var upgrades = []func(*sql.Tx) error{
 		)
 		return
 	},
+	func(tx *sql.Tx) (err error) {
```

Comment:
Needs to be the last migration in the array or this will break existing databases.